tests/contrib/zerorpc/zeropc_tests.py

```diff
@@ -36,16 +36,19 @@
 
 import pytest
 
+from elasticapm.conf.constants import ERROR
 from elasticapm.contrib.zerorpc import Middleware
 
-zerorpc = pytest.importorskip("zerorpc")
-gevent = pytest.importorskip("gevent")
 
+pytestmark = [
```

Comment:
Nice! I never figured out how to have multiple module-level marks! This will come in handy :D